test.py

```
@@ -0,0 +1,66 @@
+    time.sleep(25)
+    close_application('vlc.exe')
+    
+    #Checking for dumps generated in live_Kernel path
+    dumps_check = check_live_kernel_logs(live_kernel_path)
+
+    if dumps_check:
+        #Fetching latest dump file to validate result
+        latest_liveReport_dump = get_latest_dump(live_kernel_path)
+
+        #Step 3 : Parsing dumps generated in AmdLogAnalyzer
+        Triggering_latest_OCR_dump(latest_liveReport_dump)
```

Ruff (0.11.9)

Undefined name `Triggering_latest_OCR_dump`

(F821)